app/jobs/base.rb

```diff
@@ -102,30 +104,15 @@ def stop(exception:)
       end
 
       def self.raw_log(message)
-        @@logger ||=
-          begin
-            f = File.open "#{Rails.root}/log/sidekiq.log", "a"
-            f.sync = true
```

Comment:
I don't know why we do this manually when we can just pass the file path when initializing an instance of `Logger` and it will handle it for us.